bundle/regal/rules/testing/dubious_print_sprintf_test.rego

```diff
@@ -14,6 +14,7 @@ test_fail_print_sprintf if {
 	}`)
 
 	r := rule.report with input as module
+		with data.internal.combined_config as {"capabilities": data.regal.capabilities}
```

Comment:
This is such a common statement it might be worthwhile adding a convenience rule/func for it.

Reply:
Indeed — some test helper would be good. I have a few other ones to add, so will follow up in a future PR.